packages/imports-fs/src/index.ts

```diff
@@ -1,3 +1,11 @@
+export { Context, firstResult, Options, ResolverEngine, SubParser, SubResolver } from "@resolver-engine/core";
+export {
```

Comment:
Those explicit exports are what I called a 'nuance'. Without it, some build tool called Rollup, can't find the `gatherSources` function.